model/perf_data_test.go

```diff
@@ -278,3 +274,38 @@ func (s *perfRollupSuite) TestUpdateDefaultRollups() {
 	ts[0].Timestamp = ts[2].Timestamp
 	s.Error(result.UpdateDefaultRollups(ts))
 }
+
+func (s *perfRollupSuite) TestMergeRollups() {
+	conf, session, err := cedar.GetSessionWithConfig(s.r.env)
+	s.Require().NoError(err)
+	defer session.Close()
+
+	// without errors
+	rollups := []*PerfRollupValue{
+		{
+			Name:       "ops_per_sec",
+			Value:      50001.24,
+			Version:    1,
+			MetricType: MetricTypeThroughput,
+		},
+		{
+			Name:       "latency",
+			Value:      5000,
+			MetricType: MetricTypeLatency,
+		},
+	}
+
+	result := &PerformanceResult{}
+	s.Require().NoError(session.DB(conf.DatabaseName).C(perfResultCollection).FindId(s.r.id).One(result))
+	result.Setup(s.r.env)
+	s.NoError(result.MergeRollups(rollups))
```

Comment:
shouldn't we call this more than once to make sure that it's functionally idempotent?